testie.wren

```diff
@@ -90,18 +112,21 @@ class Testie {
             Stdout.flush()
             Fiber.new(test.fn).call()
         }
-        if (_fails > 0) Fiber.abort("Failing tests.")
+        if (_fails > 0) {
+            if (_abortAfterFailures) {
+                Fiber.abort("Failing tests.")
+            } else {
+                Process.exit(1)
```

Comment:
Do you see any reason we would need to abort at all, ever?  I think this was prior to having `Process.exit` available... I'm thinking perhaps we should just remove the branch here and always do `exit(1)`?